tokenmagic/module/settings.js

```diff
@@ -332,11 +332,11 @@ Hooks.once("init", () => {
         if (hasPresetData && data.tmfxPreset !== emptyPreset) {
             let defaultTexture = Magic._getPresetTemplateDefaultTexture(data.tmfxPreset);
             if (!(defaultTexture == null)) {
-                if (data.texture === '' || data.texture.includes('/modules/tokenmagic/fx/assets/templates/'))
+                if (data.texture === '' || data.texture.includes('./modules/tokenmagic/fx/assets/templates/'))
```

Comment:
I think that `data.texture.includes('modules/tokenmagic/fx/assets/templates/')` should also be tested. Without migration code, the world db could still contains absolute pathes.